arch/arm/boot/dts/overlays/README

```diff
@@ -235,6 +235,15 @@ Params: 24db_digital_gain       Allow gain to be applied via the PCM512x codec
                                 that does not result in clipping/distortion!)
 
 
+Name:   Aoide DAC II
```

Comment:
The name should be aoide-dacii (to match the overlay), but actually both should be aoide-dacii-audio (this is a recently adopted convention for new audio overlays).